lib/topological_inventory.rb

```diff
@@ -14,8 +14,13 @@ def self.call
 
   private_class_method def self.raw_api
     TopologicalInventoryApiClient.configure do |config|
-      config.host   = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
+      config.host = ENV['TOPOLOGICAL_INVENTORY_URL'] || 'localhost'
       config.scheme = URI.parse(ENV['TOPOLOGICAL_INVENTORY_URL']).try(:scheme) || 'http'
+      # Set up user/pass for basic auth if we're in dev and they exist.
+      if Rails.env.development?
+        config.username = ENV['DEV_TOPOLOGICAL_INVENTORY_USERNAME'] || nil
```

Comment:
@lindgrenj6 
raise an error if either of RHN_DEV_USERNAME or RHN_DEV_PASSWORD is missing.
This will allow developers to set this once in their enviornment

Reply:
@lindgrenj6 
We can remove the nil check, because with nil you will not be able to login from development environment